### DISCUSSION TITLE
Plugged in Devices go offline on a regular basis

### DISCUSSION THREAD

**Discussion by CommittotheIndian** [Koenkk/zigbee2mqtt#18793] [Devices]
I've had Z2M setup for a couple years. I have a main coordinator on a Sonoff 3.0. I also have a router setup on another Sonoff 3.0.

On occasion (and about 5 times this week) any device that is plugged in (or considered a router) will go offline. This has been happening for over a year across multiple different versions of Z2M, I'm just finally getting around to looking into it. This only affects the devices plugged in. Plugs (different brands), water shut off, the sonoff 3.0 (router). It will not affect a battery powered device. 

I then have to restart the addon to get them back online. I have not been able to find a rhyme or reason behind this issue. I've had some suggest it's interference with my Wi-Fi. Z2M is on channel 25 and no where close to any of my other Wi-Fi channels. Not to mention the battery devices still function as expected during this "blackout" of the plugged in devices so I don't think interference has anything to do with it. 

Does anyone have any advise to point me in the right direction?

**Comment by CommittotheIndian** [comment]
Anyone have any ideas on this issue?

**Comment by francisp2** [comment]
Do they really go offline ? Did you try to toggle a plug that is offline ?

**Comment by CommittotheIndian** [comment]
Yes, those light switches don't work when showing offline.
